src/functions.cpp

```diff
@@ -67,12 +68,26 @@ bool isSortedInDecreasingOrder(vector <int> vec)
 // strings
 int numberOfDigits(string str)
 {
-    return 0;
+    int counter = 0;
+
+    for (auto elem : str)
+    {
+        if (elem >= '0' && elem <= '9')
+            counter++;
+    }
+    return counter;
 }
 
 int sumOfDigits(string str)
 {
-    return 0;
+    int counter = 0;
+
+    for (auto elem : str)
+    {
+        if (elem >= '0' && elem <= '9')
+            counter = counter + elem % 10;
```

Comment:
elem % 10 - wrong
elem is a char, when we convert it to int, there are ascii code
0 - 48
1 -49
and so on

So, if input parameter is "1", function returns 9, that's wrong